src/blog/2023/08/UNS-article.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Why the Automation Pyramid blocks digital transformation - The Role of Unified Namespace
+subtitle: A Critical Examination of the Automation Pyramid's Obstruction to Digital Transformation
+description: This article analyzes the Automation Pyramid's constraints and explains the Unified Namespace, highlighting its potential to evolve digital transformation in manufacturing.
+date: 2023-08-08 
+authors: ["marian-demme"]
+#image: blog/2023/07/images/opc-ua-3/opc-ua-3-title-image.png
+tags:
+    - posts
+---
+A few years ago, I wrote an article, in German, detailing my understanding of how the Automation Pyramid, a widely adopted reference model for the IT landscape of manufacturing firms, is essentially hindering digital transformation. Now, as conversations around the Unified Namespace (UNS) and particular frameworks continue to evolve, I revisit my earlier notions, review the latest updates to reference frameworks and update my article.
```

Comment:
Do you have a link to the original article?

Reply:
Yes, but as it is in German, I don't know if it is worth linking
https://www.linkedin.com/pulse/iiot-circle-marian-raphael-demme/

Reply:
Lots of our website visitors are German speaking and also other can use Google translate if they are interested.

Reply:
See [Original article linked](https://github.com/flowforge/website/pull/1009/commits/9d64dcd2ceb488c70eeecebe0bb72350705ce963)